config/migrations/20220325092638-government-domains-add-deprecated.sparql

```diff
@@ -0,0 +1,13 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX Beleidsdomeinen: <http://themis.vlaanderen.be/id/concept-schema/f4981a92-8639-4da4-b1e3-0e1371feaa81>
+
+PREFIX gPublic: <http://mu.semte.ch/graphs/public>
+
+WITH gPublic:
+INSERT {
+    ?s owl:deprecated false .
```

Comment:
Door buggy behaviour van booleans in Virtuoso, gebruikt mu-cl-resources een eigen datatype `http://mu.semte.ch/vocabularies/typed-literals/boolean` voor booleans. Dit datatype kan dus best toegevoegd worden aan de boolean values in de migrations.
(In de praktijk gaan we er niet tegen lopen omdat de `skos:Concept`s momenteel readonly zijn en de bugs zich voornl. manifesteren bij updates van boolean values)